guides/common/modules/con_configuring-provisioning.adoc

```diff
@@ -0,0 +1,16 @@
+[id="configuring-provisioning_{context}"]
+= Configuring provisioning
+
+After your basic {Project} infrastructure is in place, you can start configuring provisioning to ensure that {Project} can seamlessly create, configure, and manage hosts.
+
+The process depends on whether you want to provision bare-metal hosts, virtual machines, or cloud instances, but it includes defining installation media, configuring provisioning templates, and other tasks.
+If you are provisioning virtual machines or cloud instances, you must also integrate your compute provider with {Project} by connecting the provider as a compute resource to {Project}.
+
+You can also automate the provisioning of your hosts by using the following methods:
+
+* Provisioning templates enable you to define the way {Project} installs an operating system on your hosts.
```

Comment:
The sentence above implies that this step is optional. But it is not; without templates, you cannot provision hosts.

Reply:
Thanks! I rephrased the sentence.